src/networking.h

```diff
@@ -23,6 +23,8 @@
 
 #include "netlink.h"
 
+#define OVS_PORT_PATH "/var/run/openvswitch/ovd_%s"
```

Comment:
Can you add a comment describing what this is (and what %s is expected to expand to).

Also, I think this should be defined as:

```
#define OVS_PORT_PATH LOCALSTATEDIR "/run/openvswitch/ovd_%s" 
```

(See `CC_OCI_RUNTIME_DIR_PREFIX`).